src/test/test_utils.cpp

```diff
@@ -14,10 +14,73 @@
 // limitations under the License.
 //*****************************************************************************
 #include "test_utils.hpp"
+
+#include "../prediction_service_utils.hpp"
+
+using tensorflow::serving::PredictRequest;
+using tensorflow::serving::PredictResponse;
+
 void waitForOVMSConfigReload(ovms::ModelManager& manager) {
     // This is effectively multiplying by 1.2 to have 1 config reload in between
     // two test steps
-    const float WAIT_MULTIPLIER_FACTOR = 1.2;
+    const float WAIT_MULTIPLIER_FACTOR = 1.1;
     const uint waitTime = WAIT_MULTIPLIER_FACTOR * manager.getWatcherIntervalSec() * 1000;
     std::this_thread::sleep_for(std::chrono::milliseconds(waitTime));
 }
+
+std::string createConfigFileWithContent(const std::string& content, std::string filename) {
+    std::ofstream configFile{filename};
+    spdlog::info("Creating config file: {}\n with content:\n{}", filename, content);
+    configFile << content << std::endl;
+    configFile.close();
+    if (configFile.fail()) {
+        spdlog::info("Closing configFile failed");
+    } else {
+        spdlog::info("Closing configFile succeed");
+    }
+    return filename;
+}
+
+ovms::tensor_map_t prepareTensors(
+    const std::unordered_map<std::string, ovms::shape_t>&& tensors,
+    InferenceEngine::Precision precision) {
+    ovms::tensor_map_t result;
+    for (const auto& kv : tensors) {
+        result[kv.first] = std::make_shared<ovms::TensorInfo>(
+            kv.first,
+            precision,
+            kv.second);
+    }
+    return result;
+}
+
+void checkResponse(const std::string outputName,
```

Comment:
Function name doesnt indicate that we require response to contain data to be +1.0.